modules/images-configuration-parameters.adoc

```diff
@@ -17,7 +17,7 @@ The `image.config.openshift.io/cluster` resource holds cluster-wide information
 
 Every element of this list contains a location of the registry specified by the registry domain name.
 
-`domainName`: Specifies a domain name for the registry. In case the registry uses a non-standard `80` or `443` port, the port should be included in the domain name as well.
+`domainName`: Specifies a domain name for the registry. To allow all subdomains, add the asterisk (`\*``) wildcard character as a prefix to the domain name (for example, `*.example.com`). If the registry uses a non-standard `80` or `443` port, the port should be included in the domain name as well.
```

Comment:
@ctauchen, we're avoiding parenthetical statements in OCP docs. Maybe?

```suggestion
`domainName`: Specifies a domain name for the registry. To allow all subdomains, add the asterisk (`\*``) wildcard character as a prefix to the domain name such as `*.example.com`. If the registry uses a non-standard `80` or `443` port, the port must be included in the domain name as well.
```

Reply:
Thanks, @jboxman. I've changed them all.